src/util-ebpf.h

```diff
@@ -40,8 +40,10 @@ struct flowv4_keys {
 		__be32 ports;
 		__be16 port16[2];
 	};
-	__u32 ip_proto;
-}  __attribute__((__aligned__(8)));
+    __u8 ip_proto:1;
```

Comment:
I always thought you are supposed to keep the type the same for those bit notation fields.

Reply:
I did not manage to find the norm but a few docs use that.